src/Moq/Obsolete/MockFactory.cs

```diff
@@ -363,8 +363,35 @@ public virtual void VerifyAll()
 		}
 
 		/// <summary>
-		/// Invokes <paramref name="verifyAction"/> for each mock 
-		/// in <see cref="Mocks"/>, and accumulates the resulting 
+		/// Calls <see cref="Mock{T}.VerifyNoOtherCalls()"/> on all mocks created by this factory.
+		/// </summary>
+		/// <seealso cref="Mock{T}.VerifyNoOtherCalls()"/>
+		/// <exception cref="MockException">One or more mocks had setups that were not satisfied.</exception>
```

Comment:
Sorry for not noticing earlier, but this should read `invocations that were not verified`, not `setups that were not satisfied`.